src/mockServer/mockServer.ts

```diff
@@ -80,6 +81,7 @@ export const mockServer = ({
         itemId: currentItemId,
         memberId: currentMemberId,
         creator: currentMemberId,
+        visibility: () => AppDataVisibility.MEMBER, // TODO: Is it right?
```

Comment:
I am not sure, but maybe you do not need the function, as you are likely to always return the same value. The others are function so their return value is given when they are called and not the value that they have when the factory is created.